src/android/ManagerService.java

```diff
@@ -207,7 +226,7 @@ public Notification createNotification(PendingIntent pendingIntent) {
                     "upload channel",
                     NotificationManager.IMPORTANCE_LOW
             );
-            NotificationManager notificationManager = (NotificationManager) ManagerService.this
+            notificationManager = (NotificationManager) ManagerService.this
```

Comment:
@sc-nick tu peux m'expliquer ce que fait cette ligne ? (Je ne comprend pas le  ```ManagerService.this``` ?

Reply:
Notifie le service des évènements qui se produisent (ManagerService.this le contexte)

Reply:
notifie le service des évènements qui se produisent (ManagerService.this le contexte)

Reply:
@sc-nick oui mais ici on peut simplement faire `this...` au lieu de `ManagerService.this...` non ? Parce qu'on est bien toujours dans la classe `ManagerService` et non pas dans une classe anonyme à l’intérieur de `ManagerService` ?
https://programming.guide/java/class-this.html

Reply:
(En même temps, ça doit être la même variable non donc c'est pas grave. Ce sont juste des questions.)